lib/ClangImporter/ClangImporter.cpp

```diff
@@ -3832,8 +3837,23 @@ void ClangImporter::Implementation::lookupValue(
     ValueDecl *decl;
```

Comment:
This is going to cause problems. Please initialize this to nullptr, otherwise, if you never set it, the condition on L1857 might have the wrong behavior (now that you're not always setting it before you read it).